### ISSUE TITLE
App - [done mod] Tasks not marked as done after marking module as done the second time

### ISSUE THREAD

**Issue by yuxuanleong** [yuxuanleong/ped#6]
### Steps to reproduce

1. Launch the app
2. Type in the command `done mod cs2109s`
3. Type in the command `add task Re-mod -c cs2109s`
4. Type in the command `done mod cs2109s`

### Expected

One of the following:
* Mark the task as done after step 4
* Error message: Unable to add task to a module that is already done

### Actual

Tasks not being marked as done.

### Screenshot
`add task Re-mod -c cs2109s`
![image.png](https://raw.githubusercontent.com/yuxuanleong/ped/main/files/4780e483-a70d-4686-96fc-54cb1310b611.png)
`done mod cs2109s`
![image.png](https://raw.githubusercontent.com/yuxuanleong/ped/main/files/3f511dc9-9a8d-47dc-bb05-c9be22ae3efa.png)
